src/Components/Selected/Selected.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+
+class Selected extends Component {
+  
+  setChecked = checked => {
+    for (let value of this.props.listOfTasks) {
+      value.isSelected = checked;
+    }
+    this.props.fromParentChangeListOfTasks(this.props.listOfTasks);
+  };
+
+  deleteSelected = () => {
+    let newTasks = [];
+    for (let value of this.props.listOfTasks) {
+      if (!value.isSelected) {
+        newTasks.push(value);
+      }
+    }
+    this.props.fromParentChangeListOfTasks(newTasks);
+  };
+
+  render() {
+    return (
+      <div className="Selected">
```

Comment:
Request!
So here className should be lover case: .select {}
And dont forget to add css class for each html element. Here you should add css class for button tags.